webapp/channels/src/components/emoji/add_emoji/add_emoji.tsx

```diff
@@ -385,6 +385,7 @@ export default class AddEmoji extends React.PureComponent<AddEmojiProps, AddEmoj
                                 />
                             </Link>
                             <SpinnerButton
+                                id='save-button'
```

Comment:
Instead of assigning an id, that has more meaning than we need and may clash with other things, we can assign `data-testid`.

That way, we can find it in jest by `wrapper.find({'data-testid': 'save-button'})`.

Reply:
Hi @larkox, requested change had been applied, kindly check